code/datums/mutations/antenna.dm

```diff
@@ -105,3 +105,22 @@
 
 /datum/mutation/human/mindreader/get_visual_indicator()
 	return visual_indicators[type][1]
+
+/datum/mutation/human/superhearing
+	name = "Super Hearing"
+	desc = "This person has superhearing and can hear incredibly quiet sounds from afar."
+	quality = POSITIVE
+	text_gain_indication = "<span class='notice'>You hear whispers and coughs.</span>"
+	text_lose_indication = "<span class='notice'>The whispers fade away, and stop.</span>"
+	instability = 30
```

Comment:
this could honestly be lowered to like 20 or maybe even 10 since this power is really marginal (for comparison, fire breathing is 30 and gives you a nerfed fireball), but that's up to you really